.gitignore

```diff
@@ -31,7 +31,6 @@ speed-measure-plugin*.json
 
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
```

Comment:
Have you removed it on purpose?

Reply:
@jerome-benoit Ok for you?

Reply:
Yes, we've done that setting with Claude.